packages/core/test-project/ios/RNEmbraceTests/RNEmbraceTests.swift

```diff
@@ -514,7 +517,44 @@ class EmbraceSpansTests: XCTestCase {
         let exportedSpans = try await getExportedSpans()
         XCTAssertEqual(exportedSpans.count, 2)
    }
+    
+    func testLogNetworkRequest() async throws {
+        module.logNetworkRequest("https://otest.com/v1/products", httpMethod: "get", startInMillis: 1723221815889, endInMillis: 1723221815891, bytesSent: 1000, bytesReceived: 2000, statusCode: 200, error: "", resolver: promise.resolve, rejecter: promise.reject)
 
+        XCTAssertEqual(promise.resolveCalls.count, 1)
+        XCTAssertTrue((promise.resolveCalls[0] as? Bool)!)
+        
+        module.logNetworkRequest("https://otest.com/", httpMethod: "POST", startInMillis: 1723221815889, endInMillis: 1723221815891, bytesSent: 1000, bytesReceived: 2000, statusCode: 200, error: "", resolver: promise.resolve, rejecter: promise.reject)
+
+        let exportedSpans = try await getExportedSpans()
+
+        XCTAssertEqual(exportedSpans[0].name, "emb-GET /v1/products")
+        XCTAssertEqual(exportedSpans[0].attributes["emb.type"]!.description, "perf.network_request")
+        XCTAssertEqual(exportedSpans[0].attributes["url.full"]!.description, "https://otest.com/v1/products")
+        XCTAssertEqual(exportedSpans[0].attributes["http.request.method"]!.description, "GET")
+        XCTAssertEqual(exportedSpans[0].attributes["http.response.body.size"]!.description, "2000")
+        XCTAssertEqual(exportedSpans[0].attributes["http.request.body.size"]!.description, "1000")
+        XCTAssertEqual(exportedSpans[0].attributes["http.response.status_code"]!.description, "200")
```

Comment:
since you're passing in startInMillis/endInMillis you could also verify the start and end times here, see `testStartSpanWithTimes`

Reply:
done